### ISSUE TITLE
Expo crashes when closing Magic login popup window

### ISSUE THREAD

**Issue by yogiben** [magiclabs/magic-js#543]
### 🐛 Description

Expo simulator crashes when Magic popup is closed e.g. by pressing "X"

### 🧩 Steps to Reproduce

1. Open popup
2. Enter bad email
3. Click X button

### 🤔 Expected behavior

Closes webview

### 😮 Actual behavior

Whole app crashes on simulator.
[error log.txt](https://github.com/magiclabs/magic-js/files/11770605/error.log.txt)

**Comment by joshuascan** [comment]
Hi @yogiben

I was unable to reproduce this issue on my end. I ran our [React Native expo demo](https://github.com/magiclabs/react-native-demo) locally, entered in an invalid email address, and was then able to close the modal successfully. I also tried with a valid email address, but then clicked the "x" before entering in the OTP, and was again able to successfully close the modal without it crashing. Are you attempting this in our demo app, or your own?
